src/Native/Runtime/unix/unixasmmacrosamd64.inc

```diff
@@ -249,6 +249,20 @@ C_FUNC(\Name):
 #define PTFF_RAX_IS_GCREF        00010000h   // iff PTFF_SAVE_RAX: set -> eax is Object, clear -> eax is scalar
 #define PTFF_RAX_IS_BYREF        00020000h   // iff PTFF_SAVE_RAX: set -> eax is ByRef, clear -> eax is Object or scalar
 
+.macro INLINE_GET_TLS_VAR
```

Comment:
Could you please add the TLS variable name as an argument to the macro, and fold it with `INLINE_GETTHREAD`?